bpf/kmesh/cgroup_sock.c

```diff
@@ -40,23 +50,38 @@ struct {
 
 static const char kmesh_module_name[] = "kmesh_defer";
 
-static inline void record_netns_cookie(struct bpf_sock_addr *ctx)
+static inline void record_netns_cookie(struct bpf_map *map, struct bpf_sock_addr *ctx)
 {
     int err;
     int value = 0;
     __u64 cookie = bpf_get_netns_cookie(ctx);
-    err = bpf_map_update_elem(&map_of_manager, &cookie, &value, BPF_NOEXIST);
+    err = bpf_map_update_elem(map, &cookie, &value, BPF_NOEXIST);
     if (err)
         BPF_LOG(ERR, KMESH, "record netcookie failed!, err is %d\n", err);
 }
 
-static inline void remove_netns_cookie(struct bpf_sock_addr *ctx)
+void record_kmesh_netns_cookie(struct bpf_sock_addr *ctx)
+{
+    BPF_LOG(DEBUG, KMESH, "record_manager_netns_cookie");
```

Comment:
nit: i think this log is meaningless, user cannot know much from. it. They still can not know which pod is managed

Reply:
> nit: i think this log is meaningless, user cannot know much from. it. They still can not know which pod is managed

I have deleted it